src/store/singleFilm/selectors.js

```diff
@@ -1,20 +1,22 @@
+/* eslint-disable consistent-return */
 const isError = state => state.singleFilm.error;
 const isLoading = state => state.singleFilm.loading;
 
-const getImages = state => state.singleFilm.response.backdrops;
-const getCasts = state => state.singleFilm.response.cast;
-const getCrew = state => state.singleFilm.response.crew;
-const getGenres = state => state.singleFilm.response.genres || false;
+const getImages = state => state.singleFilm.backdrops;
```

Comment:
You don't need these selectors at all, just get movie object from database state and get object's properties images, casts, crew etc.